README.md

```diff
@@ -1,288 +1,76 @@
-<a name="readme-top"></a>
+# Awesome-books-Project
+Basic website that allows users to add/remove books from a list. 
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-Modify this file to match your project and remove sections that don't apply.
+> Awesome-Books Web site.
+## Awesome-Books Web site Project initialization
 
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
 
-OPTIONAL SECTIONS:
-- FAQ
```

Comment:
As you can see in this comment, all of what has been listed is a required field in the README except for the FAQ section. So, I suggest that you stick to the original template, and make adjustments by adding the information related to your project.